sys/posix/sockets/posix_sockets.c

```diff
@@ -862,11 +864,15 @@ ssize_t sendto(int socket, const void *buffer, size_t length, int flags,
                                       sport, byteorder_ntohs(port));
             }
             else if (address != NULL) {
-                ipv6_addr_t local;
+                ipv6_addr_t unspec;
```

Comment:
Do you think that makes any difference? If yes, a positive one? If yes, shouldn't we modify/remove this function then?